src/test/java/org/beanpod/switchboard/fixture/DeviceFixture.java

```diff
@@ -16,14 +18,18 @@
   public static final String DISPLAY_NAME = "Device #1";
   public static final String DISPLAY_NAME2 = "Device #2";
   public static final String STATUS = "offline";
+  public static final MockMultipartFile validJsonFile =
+      new MockMultipartFile("json", "", "application/json", "{\"json\": \"someValue\"}".getBytes());
```

Comment:
Rename, since the type doesn't have to be json anymore. Value is probably fine though.

Reply:
fixed